packages/listview/docs/stories/DownloadModal.story.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { ButtonIcon } from 'react-rainbow-components';
+import { RainbowFirebaseApp } from '@rainbow-modules/app';
+import { Download } from '@rainbow-modules/icons';
+import DownloadModal from '../../src/components/DownloadModal';
+import app from '../../../../firebase';
+
+export const DefaultDownloadModal = () => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const handleDownload = async (max, format) => {
```

Comment:
max is not working here, if I select 3 as the max entries the example continues downloading all the data